Storage/docu/docu_3_code.cc

```diff
@@ -42,6 +42,16 @@ SOFTWARE.
 
 #include "../../3rdparty/gtest/gtest.h"
 
+#ifndef _MSC_VER
+DEFINE_string(client_storage_test_tmpdir,
+              ".current",
+              "Local path for the test to create temporary files in.");
+#else
+DEFINE_string(client_storage_test_tmpdir,
+              "Debug",
```

Comment:
Windows would actually be better off either using the same path as Linux/Mac or calling GetTempPath(), storing the result in a std::string, and appending "\\docu_3_code". "Debug" is a problematic path name in a typical VC++ build environment, because the actual build directories are platform first in 64 bit builds...